html/themes/custom/common_design_subtheme/components/rw-article/rw-article.css

```diff
@@ -1,3 +1,7 @@
+:root {
+  --cd-flow-space: 24px;
```

Comment:
I think this works. The main place where setting the flow value is useful is for the rivers - so the section title and the following elements `.rw-river__articles` and its children have the same vertical spacing.
In the context where we want a different value, I can define the flow value on a parent element, which as you said will be ignored by the polyfill so in IE11, the value on root will be used.

I am trying the `cd-flow` component here because all the values I mention are already 24px. I don't think variants are needed in this case.

Reply:
Ok. I would be in a favor of adding a note about that in the common_design_subtheme's README for future maintainers. Making a note for myself for now and I can do that when I do a documentation update of the modules/themes later.